atomics/T1098.001/T1098.001.yaml

```diff
@@ -0,0 +1,31 @@
+attack_technique: T1098.001
+display_name: Additional Cloud Credentials
+atomic_tests:
+- name: Create AWS Access Key and Secret Key
+  auto_generated_guid: 8822c3b0-d9f9-4daf-a043-491160a31122
+  description: |
+    Adversaries create their own new access and secret keys to programatically interact with AWS environment
+  supported_platforms:
+  - iaas:aws
+  input_arguments:
+    victim:
+      description: Username of the IAM user to create - Victim
+      type: String
+      default: "AtomicRedTeam_Victim"
+  dependencies:
+    - description: |
+        Check if the user exists before adding a user to the group
+      prereq_command: |
+        aws iam list-users | grep #{victim}
+      get_prereq_command: |
+        echo Please run atomic test T1136.003, before running this atomic
+  executor:
+    command: |
+      aws iam create-access-key --user-name #{victim} > $PathToAtomicsFolder/T1098.001/bin/aws_secret.creds
+      cd $PathToAtomicsFolder/T1098.001/bin/
+      ./aws_secret.sh
+    cleanup_command: |
+      access_key=`cat $PathToAtomicsFolder/T1098.001/bin/aws_secret.creds| jq -r '.AccessKey.AccessKeyId'`
+      aws iam delete-access-key --access-key-id $access_key --user-name #{victim}
+      rm $PathToAtomicsFolder/T1098.001/bin/aws_secret.creds
+    name: aws
```

Comment:
I think the `name` here is `sh`? (as these are not using only the aws-cli executor.